### ISSUE TITLE
US - Parts Deals - and typo in README.md

### ISSUE THREAD

**Issue by ver2go** [Makerr-Studio/CentrifugalDustSeparator-#14]
Not an issue, but wanted to share deals I found for parts. Please delete if not appropriate, or if there is a suggestion for a better place to post.

US only 

Clear PVC 6"
https://www.woodcraft.com/products/woodriver-dust-connection-clear-pvc-tube-36-x-6
Free Shipping with Code "PENCIL" $20 each
* note, scale to 99.6% seems to work for me. The scale ring indicated 100%.
** note number 2, the scale ring doesn't need walls so thick. I did a quick hack in Freecad and it cut the print time in half. Not sure if that's why it showed 100%, but I only cut out the ID of the scale ring

Festool Storage VAB-20/3
$79.60 set of 3 - Free shipping
https://www.hartvillehardware.com/product/sku-1337339/festool-dust-extractors-accessories

An Issue (typo)
https://github.com/Makerr-Studio/CentrifugalDustSeparator-/blob/main/README.md?plain=1#L48-L49
jet = yet?

**Comment by voron69-bit** [comment]
> Not an issue, but wanted to share deals I found for parts. Please delete if not appropriate, or if there is a suggestion for a better place to post.
> 
> US only
> 
> Clear PVC 6" - this is a clearance sale, so once stock runs out there will be no more https://www.woodcraft.com/products/woodriver-dust-connection-clear-pvc-tube-36-x-6 Free Shipping with Code "PENCIL" $20 each
> 
> * note, scale to 99.6% seems to work for me. The scale ring indicated 100%.
>   ** note number 2, the scale ring doesn't need walls so thick. I did a quick hack in Freecad and it cut the print time in half. Not sure if that's why it showed 100%, but I only cut out the ID of the scale ring
> 
> Festool Storage VAB-20/3 $79.60 set of 3 - Free shipping https://www.hartvillehardware.com/product/sku-1337339/festool-dust-extractors-accessories
> 
> An Issue (typo) https://github.com/Makerr-Studio/CentrifugalDustSeparator-/blob/main/README.md?plain=1#L48-L49 jet = yet?

Not sure what you're talking about the scale ring, but I've already got all my parts printed, and O rings installed. Does this 6" pvc work or not? I'm not reprinting or buying new orings.

**Comment by asucks** [comment]
> Does this 6" pvc work or not? I'm not reprinting or buying new orings.

These clearance 6" PVC tubes are the same as the original 6" PVC tubes linked.  I got the clearance ones this morning and they are identical.

**Comment by voron69-bit** [comment]
> > Does this 6" pvc work or not? I'm not reprinting or buying new orings.
> 
> These clearance 6" PVC tubes are the same as the original 6" PVC tubes linked. I got the clearance ones this morning and they are identical.

They are not. I too bought the clearance ones, and you have to reduce all the parts to 99.2%

**Comment by ver2go** [comment]
> > Does this 6" pvc work or not? I'm not reprinting or buying new orings.
> 
> These clearance 6" PVC tubes are the same as the original 6" PVC tubes linked. I got the clearance ones this morning and they are identical.
 I thought they were where I looked at the specs compared to the powertec ones. I figured they were both imported/own from the same company.

I didn't have an original powertec one to confirm. 

Did you have to scale yours? 99.6% for for me. 100% didn't on my first inletnprint test.

**Comment by asucks** [comment]
> They are not. I too bought the clearance ones, and you have to reduce all the parts to 99.2%

Maybe they are on clearance because of slight manufacturing differences?  Or maybe they are poorly cut and the edges just need to be deburred?

> Did you have to scale yours? 99.6% for for me.

What material did you print with?  

Can you guys measure the inside ID with calipers?  I have several of them so I can take some measurements soon.

**Comment by voron69-bit** [comment]
Yeah probably a manufacturer issue, since they are clearance. I print in ABS, and my printer is good out to 0.05 XYZ accuracy. So I know its not my prints. ID of this clearance tube is anywhere from 145.27 to 145.59. I could have gone SLIGHTLY larger than 99.2, but 99.3% was too snug and 99.2% fit just fine. I wasn't about to get in the weeds over a fraction of a percent.
